### ISSUE TITLE
Clean up ATMO references

### ISSUE THREAD

**Issue by tdsmith** [mozilla/firefox-data-docs#307]
ATMO delenda est: https://docs.telemetry.mozilla.org/tools/spark.html

**Comment by tdsmith** [comment]
Files to clean up:

```
tsmith-0yhv2t:tsmith src (master)$ grep -lR ATMO .
./tools/spark.md
./tools/stmo.md
./tools/advanced_analysis_with_atmo.md
./tools/interfaces.md
./SUMMARY.md
./datasets/mozetl/search_clients_daily/intro.md
./datasets/obsolete/client_count_daily/intro.md
./datasets/ping_intro.md
./datasets/other/socorro_crash/intro.md
./datasets/batch_view/main_summary/intro.md
./datasets/shield.md
./concepts/analysis_intro.md
./concepts/pipeline/data_pipeline_detail.md
./concepts/pipeline/data_pipeline.md
./cookbooks/new_ping.md
./cookbooks/parquet.md
```

**Comment by robhudson** [comment]
Since I'm working on the ATMO deprecation and job moves off of ATMO, I can assign this to myself and work on removing or updating these references.

**Comment by wlach** [comment]
Merged @robhudson's #335 which takes care of a bunch of these, but still a few remain.

**Comment by wlach** [comment]
We really should finish this off, the documentation is long enough as it is without references to decomissioned projects.
